inference-engine/src/vpu/myriad_plugin/myriad_executable_network.cpp

```diff
@@ -126,9 +135,13 @@ void ExecutableNetwork::Import(std::istream& strm,
 
     _inputInfo  = blobReader.getInputInfo();
     _outputInfo = blobReader.getOutputInfo();
-
-    _executor->allocateGraph(_device, _graphDesc, _graphBlob, blobHeader, numStages, networkName, _actualNumExecutors);
-
+    if (_inputInfo.totalSize == 0) {
+        _notConstNet = false;
+    }
+    if (_notConstNet) {
```

Comment:
It would be much better to check if the network is constant at the beginning of all methods (where it's possible), so we could do some processing regarding constant networks and `return` from the function. So all the code regarding usual cases will preserve its indentation and logic.